### PR TITLE
Pass -destination only when on Xcode 7.2

### PR DESCRIPTION
**Pull request by Mitko-Kerezov**
Keep old logic when dealing with Xcode < 7.2
In addition fix bug with detecting specific versions of Xcode - e.g. 6.3
The condition `major <= 6 && minor < 3` is not met if the Xcode version is 5.9 for example (theoretically)
Re-targeted from https://github.com/NativeScript/nativescript-cli/pull/1687
Merge after https://github.com/telerik/mobile-cli-lib/pull/645
Ping @rosen-vladimirov for a thoroguh review

### CODE REVIEW

File: lib/services/ios-project-service.ts [hunk 1]
```diff
@@ -204,10 +205,23 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
...
 				]);
+
+				let additionalArgs: string[] = [],
+					xcodeVersion = this.$xcodeSelectService.getXcodeVersion().wait();
+
+				xcodeVersion.patch = xcodeVersion.patch || "0";
+				// passing -destination apparently only works with Xcode 7.2+
+				if (xcodeVersion.major && xcodeVersion.minor &&
+					helpers.versionCompare(xcodeVersion, "7.2.0") < 0) {
+					additionalArgs = ["-arch", "i386", "VALID_ARCHS=\"i386\""];
```
Comment (rosen-vladimirov) [inline lib/services/ios-project-service.ts]:
```
'VALID_ARCHS="i386"'

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by Mitko-Kerezov** [comment]
run ci

**Comment by Mitko-Kerezov** [comment]
run ci

**Pull request merged**